meetings/backends/fedora/__init__.py

```diff
@@ -3,6 +3,7 @@
 from datetime import datetime
 from ...util import get_room_alias, time_from_timestamp
 from maubot.loader import BasePluginLoader
+from slugify import slugify
```

Comment:
Is this a core module or does it need to be added to the dependencies?

https://docs.mau.fi/maubot/dev/reference/plugin-metadata.html

(Admittedly this is not currently used by Maubot directly, but it would help us when deploying the plugin, YAML could be consumed by an Ansible playbook or similar)

Reply:
not a core module -- we could do this with a regex too i suppose, but was chose this because it is simpler and safer.

however, the fedora backend is also going to have to use a few things that are definitely not core modules (fedora-messaging specifically) so not sure if we want them as deps to the main plugin, right? (not really sure how to navigate the deps really)

Reply:
I'm a bit conflicted here, as I'm using the upstream container, so adding dependencies is not as simple as "pip install".

I think we should at least document the optional dependencies in the README?